packages/css/tailwind.config.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  prefix: 'vtmn-',
+  purge: ['src/stories/**/*.mdx'],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    screens: {
+      mobile: '0px',
+      tablet: '600px',
+      'small-desktop': '896px',
+      'medium-desktop': '1200px',
+      'large-desktop': '1800px',
+    },
+    colors: {
+      black: '#001018',
```

Comment:
Yes, this idea could be nice but because it's not documented by designers, I don't think developers will use this notation. For example, on Figma file, it will be written "brand-digital-light-2" and no "100".

Reply:
brand-digital: {
dark-1: 'var(--vtmn-color_brand-digital-dark-1)',
dark-2: 'var(--vtmn-color_brand-digital-dark-2)',
...
},

Reply:
Done :)